src/shared/ui/social-media/social-media.tsx

```diff
@@ -9,8 +12,14 @@ export type SocialMediaProps = {
 };
 
 export const SocialMedia = ({ title, href, icon }: SocialMediaProps) => (
-  <a className="social-media" href={href} target="_blank" rel="noreferrer">
+  <a
```

Comment:
Maybe <Link/>?

Reply:
decided to change it in related to links pr https://github.com/rolling-scopes/site/pull/467